package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "commercetools-sunrise-data",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "scripts": {
+    "set:authUrl": "npm config set commercetools-sunrise-data:authUrl",
```

Comment:
is it really necessary to wrap `npm config set commercetools-sunrise-data:authUrl fooBar` to then have to type `npm run-script set:authUrl fooBar`?    it's basically just obfuscation of what really happens for a few characters typing less.

Do we really need the "commercetools-sunrise-data" prefix?   just typing `npm config set apiUrl` would be the most natural way to do this

Reply:
Yes, the project name is mandatory, we could update the npm project name to something shorter and use that as a key. any idea's for a name? 

- `csd:authUrl` ?

Reply:
sunrise:authUrl  ?

Reply:
done: d7b9c29